protocol/contracts/v2/TreasuryPriceIndexOracle.sol

```diff
@@ -95,7 +95,9 @@ contract TreasuryPriceIndexOracle is ITreasuryPriceIndexOracle, TempleElevatedAc
      * @dev 18 decimal places, 1.05e18 == $1.05
      */
     function setTreasuryPriceIndex(uint96 value) external override onlyElevatedAccess {
-        uint96 _oldTpi = tpiData.currentTpi;
+        // If the cooldownSecs hasn't yet passed since the last update, then this will still
```

Comment:
This is the actual change in this commit addressing https://github.com/yAudit/temple-lending-report/issues/35

The other updates are self-identified minor updates:
* updating TPI to uint96 to avoid unnecessary casts
* fixing a couple of comments